app/views/admin/facility_groups/_form.html.erb

```diff
@@ -26,19 +26,14 @@
                         disabled: action_name == "edit" ? true : false %>
       <% end %>
 
-      <% if facility_group.facilities.present? %>
-        <%= form.collection_check_boxes :facility_ids, facility_group.facilities, :id, :name, label: "Associated facilities", checked_value: 1 %>
-      <% end %>
-
-      <% if Facility.where(facility_group: nil).present? %>
-        <%= form.collection_check_boxes :facility_ids, Facility.where(facility_group: nil), :id, :name, label: "Unassociated facilities", checked: true %>
-      <% else %>
-        <% if facility_group.new_record? %>
-          <div class="row">
-            <div class="col-md-9 mb-3 text-warning">All existing facilities belong to a facility group.</div>
-          </div>
+      <label>Associated Facilities</label>
+      <%= content_tag(:ul, class: "pl-3") do %>
+        <% facility_group.facilities.each do |facility| %>
+          <li>
+            <%= link_to facility.name, admin_facility_group_facility_path(facility_group, facility), target: :_blank %>
+          </li>
         <% end %>
-      <% end %>
+      <% end if facility_group.facilities.present? %>
```

Comment:
Could we just 

```
<% if facility_group.facilities.present? %>
  <ul>
...
```

Not sure this nifty clause saves us much codes, what do you think?

Reply:
Ha, I dislike trivial conditionals blocking UI elements – makes the markup hard to read. But... changed (don't feel strongly).